data/locale/sv.ts

```diff
@@ -1063,7 +1063,7 @@ If you&apos;re interested in translating LMMS in another language or want to imp
     </message>
     <message>
         <source>FREQ</source>
-        <translation>FREKV.</translation>
+        <translation>FREK</translation>
```

Comment:
Either this or just revert to no translation here.

Reply:
I reverted those to Freq/FREQ and some other Instrument plugin knob texts to the English original for now.